e2e/Makefile

```diff
@@ -152,3 +157,74 @@ setup:
 	chmod a+x $(BINDIR)/kubectl $(BINDIR)/kind
 
 .PHONY: launch-kind shutdown-kind start-lvmd stop-lvmd test clean setup
+
+.PHONY: daemonset-lvmd/create-vg
+daemonset-lvmd/create-vg:
+	mkdir -p build $(BACKING_STORE)
+	go build -o build/lvmd ../pkg/lvmd
+	if [ -f $(BACKING_STORE)/backing_store_lvmd* ]; then $(MAKE) sdaemonset-lvmd/remove-vg; fi
+
+	mkdir -p /tmp/topolvm/worker_lvmd;
+	mkdir -p /tmp/topolvm/lvmd_lvmd;
+	truncate --size=20G $(BACKING_STORE)/backing_store_lvmd_1;
+	$(SUDO) losetup -f $(BACKING_STORE)/backing_store_lvmd_1;
+	$(SUDO) vgcreate -y node-lvmd-myvg1 $$($(SUDO) losetup -j $(BACKING_STORE)/backing_store_lvmd_1 | cut -d: -f1);
```

Comment:
Is there any strong reason to use node-lvmd-myvgXX for vg names? Just using node-myvg1 as non-daemonset tests looks simpler.